packages/table/components/Table.tsx

```diff
@@ -122,6 +138,47 @@ export class Table<T> extends React.PureComponent<TableProps, TableState> {
     );
   }
 
+  private fillTableWidth(
+    colWidths: number[],
+    remainingWidth: number,
+    width: number
+  ) {
+    let filledWidth;
+    const totalWidth = colWidths.reduce((acc, curr) => acc + curr, 0);
+    const widthToFill = width - totalWidth;
+    const columns = React.Children.toArray(this.props.children) as Array<
+      React.ReactElement<ColumnProps>
+    >;
+    const fillWidthColsIdxArr: number[] = columns
+      .filter(child => child.props.growToFill)
+      .map(col => columns.indexOf(col));
+    const fillColumns = (colsIdxArr, widthToFill) => {
+      if (widthToFill <= 0) {
+        return;
+      }
+      colsIdxArr.forEach(colIdx => {
+        const oldWidth = colWidths[colIdx];
+        const newWidth = widthToFill / fillWidthColsIdxArr.length + oldWidth;
+
+        if (newWidth <= oldWidth) {
+          return;
+        }
+
+        colWidths[colIdx] = clampWidth(
```

Comment:
we might want to move the function to the same utility module as well. That may not be a separate file. We could totally have those function in the Table.tsx and exporting them so that they could be tested.